addons/event/models/event.py

```diff
@@ -149,44 +151,58 @@ class EventEvent(models.Model):
     # Registration fields
     registration_ids = fields.One2many(
         'event.registration', 'event_id', string='Attendees',
-        readonly=False, states={'done': [('readonly', True)]})
+        readonly=False)
     # Date fields
     date_tz = fields.Selection('_tz_get', string='Timezone', required=True, default=lambda self: self.env.user.tz or 'UTC')
     date_begin = fields.Datetime(
         string='Start Date', required=True,
-        tracking=True, states={'done': [('readonly', True)]})
+        tracking=True)
     date_end = fields.Datetime(
         string='End Date', required=True,
-        tracking=True, states={'done': [('readonly', True)]})
+        tracking=True)
     date_begin_located = fields.Char(string='Start Date Located', compute='_compute_date_begin_tz')
     date_end_located = fields.Char(string='End Date Located', compute='_compute_date_end_tz')
     is_one_day = fields.Boolean(compute='_compute_field_is_one_day')
+    duration_str = fields.Char(compute='_compute_duration_str')
+
+    kanban_state = fields.Selection([('normal', 'In Progress'), ('done', 'Done'), ('blocked', 'Blocked')])
+    kanban_state_label = fields.Char(compute='_compute_kanban_state_label', string='Kanban State Label', tracking=True, store=True)
+
+    legend_blocked = fields.Char(related='stage_id.legend_blocked', string='Kanban Blocked Explanation', readonly=True, related_sudo=False)
```

Comment:
@tde-banana-odoo I need them, otherwise, I do not have the right label in the form view
![_tmp_screenshot](https://user-images.githubusercontent.com/54316226/71412302-6fe77b00-264d-11ea-9efd-7d101555809f.png)

Reply:
Ah yes hardcoded kanban state behavior ... keep the related then. However the computed field should be based on stage values, not the one not stored coming from the event, to avoid dependencies issues.

Reply:
@tde-banana-odoo I nned thoose 3 fields to have the right labels for the kanban state in the form view :/

![_tmp_screenshot](https://user-images.githubusercontent.com/54316226/71412910-0ae15480-2650-11ea-9c97-6764cd9ffb31.png)